Create ui-c8y-1023-53-1-sso-users-confirm-with-code-when-suspend-or-delete-tenant.md (#4370)
SoftwareAG/c8y-docs: content/change-logs/platform-services/ui-c8y-1023-53-1-sso-users-confirm-with-code-when-suspend-or-delete-tenant.md

---
date: ""

> title: SSO tenant management improved when deleting or suspending a tenant

Reviewer: ```suggestion
title: SSO tenant management improved when suspending or deleting a tenant
```

> title: SSO users can now confirm tenant suspension or deletion with a code instead of a password

Reviewer 2: ```suggestion
title: SSO tenant management improved when deleting or suspending a tenant
```

product_area: Platform services
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-0UgqXH1Ys
    label: Administration
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-66119
version: 1023.53.1
---

> When suspending or deleting a tenant in an SSO-enabled environment, users were previously prompted to enter their Cumulocity account password to confirm the action. However, SSO users authenticate through an external identity provider and do not have a Cumulocity password, making it impossible for them to complete this confirmation step. Now, instead of requiring a password, SSO users receive a randomly generated code that they can enter to confirm the suspension or deletion of a tenant. This change ensures that SSO users can perform these critical tenant management operations without friction, while maintaining the same level of security through code-based confirmation as password-based confirmation provides for non-SSO users.

Reviewer 2: ```suggestion
When suspending or deleting a tenant in an SSO-enabled environment, the behavior has been changed from requiring a password to providing a randomly generated code that is entered in order to complete the confirmation step. This change ensures that SSO users can perform these critical tenant management operations without friction, while maintaining the same level of security through code-based confirmation as password-based confirmation provides for non-SSO users.
```
